PureBasicIDE/Common.pb

```diff
@@ -2865,4 +2865,22 @@ CompilerIf #PB_Compiler_Debugger
 CompilerEndIf
 
 UseMD5Fingerprint()
-UseCRC32Fingerprint()
\ No newline at end of file
+UseCRC32Fingerprint()
+
+; Limit ResizeGadget
+
+Procedure ResizeGadgetLimit(Gadget, x, y, Width, Height)
```

Comment:
I think such hacks do not really belong in the IDE. This should be handled in the ResizeGadget() command directly if it does not work properly otherwise.

The IDE should not contain fixes for bugs in PB commands.

If it is to stay here then I think it should go into the `Macro.pb` file instead.

Reply:
I wanted to remove the hack again.
Not even as a macro. Better customise it where ResizeGadget is called.